query.go

```diff
@@ -281,18 +281,14 @@ func getUserVideoDriverPattern(
 func newVideoSourceFromDriver(
 	videoDriver driver.Driver,
 	mediaProp prop.Media,
-	logger golog.Logger,
 ) (MediaSource[image.Image], error) {
 	recorder, ok := videoDriver.(driver.VideoRecorder)
 	if !ok {
 		return nil, errors.New("driver not a driver.VideoRecorder")
 	}
 
 	if driverStatus := videoDriver.Status(); driverStatus != driver.StateClosed {
-		logger.Warnw("video driver is not closed, attempting to close and reopen", "status", driverStatus)
-		if err := videoDriver.Close(); err != nil {
-			logger.Errorw("error closing driver", "error", err)
-		}
```

Comment:
The driver may be in use by another camera component. Therefore, we shouldn't close it if it's in use.